frontend/src/components/LoginPage/LoginPage.css

```
@@ -0,0 +1,141 @@
+#loginBox {
+    height: 450px;
+    width: 350px;
+    background: #FEFEFE;
+    border: 0.5px solid #C4C4C4;
+    margin: auto;
+    margin-top: 18vh;
+    box-shadow: 0px 10px 24px rgba(49, 77, 137, 0.15);
+    text-align: center;
+    padding: 40px 80px;
+}
+
+#loginLogo {
+    height: 115px;
+    width: auto;
+}
+
+#loginForm {
+    margin-top: 30px;
+}
+
+.loginInput {
+    height: 30px;
+    font-size: 15px;
+    width: 95%;
+    border: 1.5px solid rgba(49, 77, 137, 0.6);
+    padding: 2%;
+}
+
+.loginLabel {
+    color: #314D89;
+    text-align: left;
+    margin: 0 0 2px 0;
+    font-weight: 500;
+    font-size: 15px;
+}
+
+#loginPassword {
+    margin-top: 30px;
+    display: flex;
+    justify-content: space-between;
+}
+
+#loginForgotPassword {
+    color: #6DB340;
+    margin: 0;
+    font-weight: 500;
+    font-size: 13px;
+}
+
+.loginForgotPassword:hover {
```

```suggestion
#loginForgotPassword:hover {
```
Just a small typo that caused this hover feature to not work